controllers/kyma_controller_test.go

```diff
@@ -43,13 +41,22 @@ var _ = Describe("Kyma with deprecated Condition", Ordered, func() {
 			Message:            "Deprecated",
 			LastTransitionTime: metav1.Time{Time: time.Now()},
 		})
-		kyma.ManagedFields = nil
-		Eventually(controlPlaneClient.Patch(ctx, kyma, client.Apply,
-			client.FieldOwner(v1beta1.UnmanagedKyma)), Timeout*2, Interval).Should(Succeed())
-		By("having transitioned the CR State to Ready as there are no modules")
-		Eventually(CheckKymaConditions(ctx, controlPlaneClient, kyma.GetName(),
-			[]v1beta1.KymaConditionType{v1beta1.ConditionTypeModules}),
-			Timeout, Interval).Should(BeTrue())
+		kyma.Status.Conditions = append(kyma.Status.Conditions, metav1.Condition{
```

Comment:
This test is fine, but in this test, you only test the `InitConditions`, which can be written with a simple unit test, and doesn't need to be based on Ginkgo framework.

You can reference this [unit test example](https://github.com/kyma-project/lifecycle-manager/blob/main/internal/manifest/v1beta1/spec_resolver_test.go) to check how to write a unit test, and please also reference this [goland test guide](https://www.jetbrains.com/help/go/testing.html), it offers us a feature to automatically generate good unit test template to unit test the functions.

Reply:
But Ginkgo is a general framework for testing in Golang. I think it is better to stick to one framework for consistency